src/wazuh_modules/syscollector/src/syscollectorImp.cpp

```diff
@@ -16,6 +16,11 @@
 #include "hashHelper.h"
 #include "timeHelper.h"
 
+constexpr auto MAX_DELAY_TIME
+{
+    300
+};
```

Comment:
Use std::chrono::second
```suggestion
constexpr std::chrono::second MAX_DELAY_TIME
{
    300
};
```